### ISSUE TITLE
Position class disappear when container applied

### ISSUE THREAD

**Issue by yeungc** [ben-rogerson/twin.macro#53]
Hello, the position class seems cannot place together with container.

Without container:
```
const WebNav = tw.nav`relative flex justify-between items-center mx-auto`
```
The position class is applied:
![1](https://user-images.githubusercontent.com/7931068/80303397-a511fe80-87e2-11ea-855d-dbe8a8a5206c.png)

With container:
```
const WebNav = tw.nav`relative container flex justify-between items-center mx-auto`
```
The position class disappear:
![2](https://user-images.githubusercontent.com/7931068/80303361-7ac04100-87e2-11ea-870c-ae10db88306a.png)

**Comment by ben-rogerson** [comment]
Thanks for the report.
There will be a bugfix in the next version due within the next week 👍

**Comment by ben-rogerson** [comment]
Fixed in 1.0.0 🎉
